app/assets/stylesheets/application.scss

```diff
@@ -46,3 +46,23 @@ p {
   height: 1px;
   overflow: hidden;
 }
+
+.hide-bullet-points {
+  list-style: none;
+}
+
+.hide-bullet-points li{
+  margin: 5px 0;
+}
+
+.mobile-display-only{
+  color: white !important;
+  font-weight: bold;
+  margin-top: 20px;
+}
+
+@media (min-width: 48.0625em){ 
+  .mobile-display-only{
```

Comment:
I think it would be nicer to have the class describing what is being hidden, looks like a "Menu heading thing".

Reply:
Resolved in 91ff03d